src/I18n/DateFormatTrait.php

```diff
@@ -85,6 +96,36 @@ public static function setDefaultLocale($locale = null)
         static::$defaultLocale = $locale;
     }
 
+    /**
+     * Gets whether locale format parsing is set to lenient.
+     *
+     * @return bool
+     */
+    public static function lenientParsingEnabled(): bool
```

Comment:
The typehints are not possible in PHP5.6+

Reply:
My fault